charter.md

```diff
@@ -1,29 +1,34 @@
 # Interop Team Charter
 
-The Interop Team exists to make the web more interoperable in key areas, prioritized by web developer and user needs.
+The Interop Team exists to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
 
-TODO: Why this is important, background with MDN DNA etc.
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
 
 ## Scope
 
-TODO: Roughly, improving the interoperability of any "observable" web platform behavior.
+The team works by prioritizing specific web platform features. The minimum requirements for a feature to be in scope are:
 
-TODO: Not writing specs as part of this team, but can work within regular working groups to get stuff done as necessary.
+- The feature is defined, at least partially, by a specification developed by an organization like the W3C, WHATWG, TC39, Khronos, etc.
+- The feature can be tested in a test suite like [web-platform-tests](https://github.com/web-platform-tests/wpt), [Test262](https://github.com/tc39/test262), etc.
 
-## Governance
+Browser features that don't affect web developers, like bookmarks, are out of scope.
+
+Writing specifications is not in scope, but the team can liaise with and work within other working groups to make progress in key areas.
 
-The team will meet regularly, with agenda issues created in the repo at least 48 hours in advance, and meeting minutes posted in the issue.
+## Governance
 
 The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition.
```

Comment:
What defines a participating organization?

Is it "track record of contributions"? https://github.com/web-platform-tests/interop/pull/102/files/274a676f9dbff051cea325f652e067821487f88b..0cee103e550562b2cfa44ab6d8788a9e0ad599e5#diff-d22b3d6550c34594a6e2d6b2148ed17907c5d598b10f407b6fbecf07bf25ea97R34

Reply:
It's the list of organizations in https://github.com/web-platform-tests/interop/blob/main/supporters.md, who have in some sense "joined" the team. We should write down something about how to join, and there a "track record of contributions to the web platform" is the criteria we've discussed.